scripts/build_gh_pages

```diff
@@ -29,25 +29,38 @@ if [[ -n "$(ls "${TARGET_DIR}"/*)" ]]; then
   exit 1
 fi
 
-# Generate docs directly in the target dir.
-cargo doc --no-deps --target-dir="${TARGET_DIR}"
+readonly CURRENT_DIR="$(pwd)"
 
-# Remove non-doc artifacts from the target dir.
-rm --recursive --force "${TARGET_DIR}/debug"
+paths=("." "oak_abi" "oak_utils")
+names=("oak" "oak_abi" "oak_utils")
+titles=("SDK" "ABI" "Utils")
 
-# Remove non-deterministic files.
-rm "${TARGET_DIR}/.rustc_info.json"
+for i in {0..2}; do
+  cd "${CURRENT_DIR}"
+  target="${TARGET_DIR}/${names[$i]}"
+  mkdir -p target
+  cd "${paths[$i]}"
 
-# The docs generated from the Cargo workspace do not include a workspace-level index, so we generate
-# one here and redirect to the Oak SDK documentation.
-cat <<-END > "${TARGET_DIR}/index.html"
+  # Generate docs directly in the target dir.
+  cargo doc --no-deps --target-dir="${target}"
```

Comment:
(pre-existing) If this script were run twice on the same machine, is there any mechanism here to remove old files that are no longer in the repo but were left over in the target directory from a previous run?